components/MainNavBar.vue

```diff
@@ -65,7 +65,7 @@ export default {
                  { name : 'Resources', url : '/resources', external: false },
                  { name : 'Contact', url : '/contact', external: false },
             ] },
-            { name : 'Blog', url : ' https://blogs.ed.ac.uk/ecarroll3_witchcraft_visualisation/', external: true, subpages : [] }
+            { name : 'Blog', url : ' https://blogs.ed.ac.uk/witchcraft_visualisation/', external: true, subpages : [] }
```

Comment:
Hey Josep, Can you remove the extra space in front of the url so should be 'https:// and not ' https://

Sorry, realize the original was like that, but the url should not have the preceding space.